Makefile

```diff
@@ -40,11 +42,48 @@ update-tools:
 run-etcd:
 	@-docker rm -f calico-etcd
 	docker run --detach \
-	-p 2379:2379 \
+	--net=host \
 	--name calico-etcd quay.io/coreos/etcd:v2.3.6 \
-	--advertise-client-urls "http://127.0.0.1:2379,http://127.0.0.1:4001" \
+	--advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379,http://$(LOCAL_IP_ENV):4001,http://127.0.0.1:4001" \
 	--listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001"
 
+run-kubernetes-master: stop-kubernetes-master run-etcd
```

Comment:
Should this have a  ## comment so that it appears in the make target list, or are you keeping it hush hush?  Same for the stop target.